src/sentry/conf/server.py

```diff
@@ -1841,6 +1841,7 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     # "org:superuser",  Do not use for any type of superuser permission/access checks
     # Assigned to active SU sessions in src/sentry/auth/access.py to enable UI elements
     "member:read",
+    "member:invite",
```

Comment:
nit: keep read, write admin ordering
```suggestion
    "member:invite",
    "member:read",
```